examples/text_classification/train.py

```diff
@@ -118,7 +118,7 @@ def test(data_):
         description='Train a text classification model on text classification datasets.')
     parser.add_argument('dataset', choices=text_classification.DATASETS)
     parser.add_argument('--num-epochs', type=int, default=5,
-                        help='num epochs (default=3)')
+                        help='num epochs (default=5)')
```

Comment:
Any reason to change the number of epochs? If not, I would suggest to keep the original one.

Reply:
The actual default is 5 (line above), however in the help string it says it's 3.